package.json

```diff
@@ -13,7 +13,7 @@
   },
   "author": "ACME Inc.",
   "scripts": {
-    "test": "./test/spec.rb",
+    "test": "mocha test/*test.js",
```

Comment:
Assuming the filenames will end with `test` is already too much for a boilerplate in my opinion. I would simple glob everything in the `test` folder. Any file that does not contain any test for Mocha to run (such as `utils.js`) will be safely discarded.